SECURITY.md

```diff
@@ -1,32 +1,56 @@
-# Security
+# Coordinated Vulnerability Disclosure Policy
 
-> **IMPORTANT**: If you find a security issue, you can contact our team directly at
-security@interchain.berlin, or report it to our [bug bounty program](https://hackerone.com/cosmos) on HackerOne. *DO NOT* open a public issue on the repository.
+The Cosmos ecosystem believes that strong security is a blend of highly technical security researchers
+who care about security and the forward progression of the ecosystem and the attentiveness and
+openness of Cosmos core contributors to help continually secure our operations.
 
-## Bug Bounty
+> **IMPORTANT**: *DO NOT* open public issues on this repository for security vulnerabilities.
 
-As part of our [Coordinated Vulnerability Disclosure Policy](https://tendermint.com/security), we operate a
-[bug bounty program](https://hackerone.com/cosmos) with Hacker One.
+## Scope
 
-See the policy linked above for more details on submissions and rewards and read
-this [blog post](https://blog.cosmos.network/bug-bounty-program-for-tendermint-cosmos-833c67693586) for the program scope.
+| Scope                 |
+|-----------------------|
+| last release (tagged) |
```

Comment:
Should we open for scope all the actively maintained release lines and not only the latest?

Reply:
Updated it.